sabnzbd/downloader.py

```diff
@@ -482,6 +482,10 @@ def run(self):
 
             if readkeys or writekeys:
                 read, write, error = select.select(readkeys, writekeys, (), 1.0)
+                
+                # Why check so often when so few things happend?
+                if len(read) < len(readkeys)/4:
+                    time.sleep(0.01)
```

Comment:
I'm surprised that this helps.
I would have thought that more sleep was needed here.